grpc/src/main/java/com/linecorp/armeria/common/grpc/GrpcJsonUtil.java

```diff
@@ -59,7 +63,13 @@ public static MessageMarshaller jsonMarshaller(
             jsonMarshallerCustomizer.accept(builder);
         }
 
-        return builder.build();
+        try {
+            return builder.build();
+        } catch (RuntimeException e) {
+            logger.warn("Failed to instantiate a json marshaller for {}." +
+                        " Consider using GrpcJsonMarshaller.ofGson instead.", methods);
```

Comment:
How about using FQCN and leaving the cause?
```suggestion
                        " Consider using {}.ofGson() instead.", GrpcJsonMarshaller.class.getName(), methods, e);
```